### DISCUSSION TITLE
怎么实现copilt本地化部署

### DISCUSSION THREAD

**Discussion by revilhui** [community/community#127487] [Copilot Conversations]
### Select Topic Area

Question

### Body

怎么实现copilt本地化部署

**Comment by Akash1134** [comment]
Hi @revilhui , 

Thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
